zips/zip-0317.rst

```diff
@@ -75,6 +95,9 @@ Requirements
   two inputs and two outputs for each shielded pool used by the transaction).
 * Users should be able to spend a small number of UTXOs or notes with value
   below the marginal fee per input.
+* Post the upgrade to the ZSA protocol, users should be discouraged from issuing new “garbage” custom Assets.
+* Post the upgrade to the ZSA protocol, users should be discouraged from finalizing Assets frivolously.
```

Comment:
Does this really need to be a requirement? A DoS attack (whether intentional or otherwise) that only had to pay a significant "create new asset" fee, would still have the same order of magnitude expense as one that had to also pay a significant "set finalize flag" fee.

I don't see what a DoS adversary would gain from finalizing the assets they created, anyway (unless there is some implementation bug associated with finalization, and it seems no more likely there would be a bug there than anywhere else).

Reply:
This was an older iteration, sorry. I've removed the finalization fee and kept only an asset "creation" fee as we discussed last week.